File: DSA_Python/Basic_Maths/Count_Digits.py
def countdigits(n):
    num = n;
    count = 0;
    while n > 0:
        last = n % 10;
        if num % last == 0:
            count += 1;
        n = n // 10;
    return count;


if __name__ == "__main__":
    print(countdigits(12))

File: DSA_Java/Basic_Maths/src/Count_Digits.java
public class Count_Digits {

    public static int count(int n){
        int count = 0;
        int num = n;
        while(n > 0){
            if(n%num == 0){
                count++;
            }
            n = n/10;
        }
        return count;
    }

    public static void main(String[] args){
        System.out.println(count(12));
    }
}
